app/components/Dashboard.tsx

```diff
@@ -81,13 +81,6 @@ export function Dashboard({
     databaseGroups,
   };
 
-  const readyForTesting = {
-    ...deploymentProps,
-    deployments: state.deployment.filter(({ branch }) =>
-      branchesWithoutPullRequests.includes(branch)
```

Comment:
do not remove this - please revert the changes in all files except for siteConfig.ts

reason: readyForTesting is not exactly the same as automatic deployments
ready for testing means deployments on branches on which there is also a PR that is ready to be tested - regardless of whether that deployment was done automatically or not